include/extractor/extractor_config.hpp

```diff
@@ -84,14 +83,12 @@ struct ExtractorConfig final : storage::IOConfig
     std::vector<boost::filesystem::path> location_dependent_data_paths;
     std::string data_version;
 
-    unsigned requested_num_threads;
-    unsigned small_component_size;
+    unsigned requested_num_threads = 0;
+    unsigned small_component_size = 1000;
 
-    bool generate_edge_lookup;
-
-    bool use_metadata;
-    bool parse_conditionals;
-    bool use_locations_cache;
+    bool use_metadata = false;
+    bool parse_conditionals = false;
+    bool use_locations_cache = true;
```

Comment:
Values are chosen as default values we use in [program_options](https://github.com/Project-OSRM/osrm-backend/blob/41dda32546399f1dc12af1de41668993de44c7dc/src/tools/extract.cpp#L66).